tests/unit/migrations/namespace-form-fields.js

```diff
@@ -4,6 +4,39 @@ var sinon = require('sinon'),
     utils = require('../utils'),
     migration = require('../../../migrations/namespace-form-fields');
 
+// Arguments : docArrayForFirstBatch, docArrayForSecondBatch, ...
+var makeStubs = function() {
```

Comment:
This looks like a job for ES6!

You want varargs right?

```js
const makeStubs = (...batches) => {
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator